### DISCUSSION TITLE
question 3

### DISCUSSION THREAD

**Discussion by Maxi2022gt** [XeroOl/notitg-mirin#82] [General]
how do i import shaders?
do i have to import shadertoy shaders ?

**Comment by XeroOl** [comment]
Again, not part of the mirin template.

You can put Frag= or Vert= property on an actor in the XML
Generally, NotITG shaders won't be compatible with shadertoy, partly because shadertoy's variable names are a bit different, and partly because shadertoy uses a newer glsl version, while NotITG is on old glsl.

**Reply by Maxi2022gt** [reply]
so what platform do i put shaders?

**Reply by XeroOl** [reply]
I think the best advice for shaders is to look at how other modfiles use shaders.
`#version 120` is a common target.

**Reply by Maxi2022gt** [reply]
i already learned how to use shaders.
